README.md

```diff
@@ -35,4 +35,7 @@ It can be pulled with `docker pull bpfk/stoa`.
 ## Manual
 
 One can use `npm start` to run Stoa, and access it on its default port (3836).
+```sh
+$ npm start -- -c [configFile]
```

Comment:
I think it would be nice to introduce `config.example.yaml`.
But the port here is 4242.

Reply:
Yes, It can do it easily from the perspective of the user side.